databases/foundationdb/Portfile

```diff
@@ -5,28 +5,51 @@ PortGroup           github 1.0
 PortGroup           cmake 1.1
 PortGroup           compiler_blacklist_versions 1.0
 
-github.setup        apple foundationdb 6.3.12
-revision            1
+github.setup        apple foundationdb 6.3.24
+revision            0
```

Comment:
I believe the `revision` should be incremented with each change to the Portfile.

Reply:
On second thought, the version increase should be sufficient and thus the change of `revision` to `0` is fine :)